### DISCUSSION TITLE
Manually trigger a reading via MQTT cmd

### DISCUSSION THREAD

**Discussion by RasputinBerry** [jomjol/AI-on-the-edge-device#1410] [Ideas]
It would be great if you could trigger the water level manually...
Example: The bathroom door is opened, you do your (water) activities and when you go out or close the door, you get back the amount of water you used. For this you would have to create 2 raw values before and after.

**Comment by caco3** [comment]
That is a request we had before.

I understand your idea, but what do you do if eg.  the washing machine is running at the same time or somebody else is cooking/showering/using another toilet? It would be almost impossible to take this into consideration!

**Reply by caco3** [reply]
Feature request for this: https://github.com/jomjol/AI-on-the-edge-device/issues/1351

**Comment by RasputinBerry** [comment]
That's right. But I didn't want to include all eventualities. The main thing is that you have the trigger and the 2 objects in MQTT that you can easily access. It's up to you to decide what to do with it...

I found this command by accident: `http://IP/wasserzaehler.html`. It already provides a current value and could be processed via Blockly. I'll try that...

Another question:
Is there a complete overview of all commands somewhere in this project?

**Reply by caco3** [reply]
I think you are looking for https://github.com/jomjol/AI-on-the-edge-device/wiki/REST-API

**Comment by RasputinBerry** [comment]
> I found this command by accident: http://IP/wasserzaehler.html. It already provides a current value and could be processed via Blockly. I'll try that...

... the above value is "only" the current value and not a **current triggered** value, right? Then that's not what I want... ...I could have figured that out before ...

**Reply by caco3** [reply]
Not sure what you mean with "current triggered" value. `wasserzaehler.html` as well as `value` return the numbers of the last round.

**Comment by RasputinBerry** [comment]
> .... the numbers of the last round.

Correct, but that is the last value triggered by the "AutoTimer Interval" stored in "config.ini" and not by an independent trigger...

**Reply by caco3** [reply]
there is, as of now, no independant trigger!
